queries.yaml

```diff
@@ -328,13 +328,13 @@ post_process:
   - fn: TileStache.Goodies.VecTiles.transform.drop_features_where
     params:
       source_layer: landuse
-      start_zoom: 13
+      start_zoom: 6
       where: >-
         properties.get('mz_is_building') is True
   - fn: TileStache.Goodies.VecTiles.transform.drop_properties
     params:
       source_layer: landuse
-      start_zoom: 13
+      start_zoom: 6
```

Comment:
> Yes. More specifically any time we update the starting zoom in the landuse query for when we include the mz_is_building property.

I can't find it now, but at some point I think @zerebubuth had mentioned that we might want to just skip these targeted removals, and just have a single transform at the end that removes all `mz_*` properties from all features. That seems more universal and more robust to me.